### PR TITLE
Completed html-syntax

### PR DESCRIPTION
**Pull request by Aiden-Peace300**
image

### CODE REVIEW

File: html-syntax/html-syntax-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Give five examples of HTML element types.
+
+- What is the purpose of HTML attributes?
+
+- Give an example of an HTML entity (escape character).
```
Comment (shawnkost) [inline html-syntax/html-syntax-quiz-notes.md:11]: Don't forget quiz questions

**Review by shawnkost — CHANGES_REQUESTED** [review]
Don't forget the quiz questions

File: html-syntax/html-syntax-quiz-notes.md (hunk 1 above)
Comment (Aiden-Peace300) [reply html-syntax/html-syntax-quiz-notes.md]: updated

**Review by shawnkost — APPROVED** [review]
👍🏻

**Comment by Aiden-Peace300** [comment]
image
image

**Pull request merged**